### DISCUSSION TITLE
Trying to get 2FA working in Portainer

### DISCUSSION THREAD

**Discussion by jump** [mrlt8/docker-wyze-bridge#342] [Q&A] [answered]
Can't find the right place to put the commands in, I'm fairly new to Portainer, but like it. 

Can anybody help?

I can't seem to figure out where to echo the token out to, while also using portainer. 

I was following along here originally: 

https://pythonawesome.com/rtmp-rtsp-hls-bridge-for-wyze-cams-in-a-docker-container/#Multi-Factor-Authentication

**Comment by mrlt8** [comment, marked as answer]
Try going to the container details and looking for the console. You should be able to use the command: `echo "123456" > /tokens/mfa_token`

Edit: updated the wiki with 2FA info for Portainer:
https://github.com/mrlt8/docker-wyze-bridge/wiki/Portainer

**Reply by jump** [reply]
that was it! hardest part was finding the actual console icon to click.
